tests/e2e/workload-lifecycle/services/dotnet-http-server/obj/dotnet-http-server.csproj.nuget.dgspec.json

```diff
@@ -1,20 +1,20 @@
 {
   "format": 1,
   "restore": {
-    "/app/dotnet-http-server.csproj": {}
+    "/Users/amirblum/repos/odigos/tests/e2e/workload-lifecycle/services/dotnet-http-server/dotnet-http-server.csproj": {}
```

Comment:
did you mean to commit all these extra files?

Reply:
fixed